CHANGELOG.md

```diff
@@ -2,8 +2,15 @@
 
 This file is used to list changes made in each version of the PHP cookbook.
 
-## Unreleased
-
+## 8.0.0 (2020-07-09)
+
+- Drop support for:
+  - Debian 8
+  - Ubuntu < 16.04
+  - Amazon Linux 1
+- Set default PHP version to 7.2.x
+- Update Ubuntu 18.04 support to source build PHP 7.2
```

Comment:
7.2 is technically [EOL'd in November](https://www.php.net/supported-versions.php), we may want to default to 7.4 where possible.

Reply:
The reasoning behind this is that 18.04’s package installs 7.2, so all of the attributes are meant to align with that.

I hear you on the EOL coming up soon; any idea if 18.04 is gonna get a bump to its package versions?

Reply:
Ah good point, then this should be OK for now. We can see what happens later I guess.